packages/configuration-validation/test/lib.test.js

```diff
@@ -70,14 +76,76 @@ describe('Configuration Validation', () => {
 
       expect(buildConfigObject(passedConfig)).toEqual({
         clientId: '{clientId}',
+        client_id: '{client_id}',
         issuer: '{issuer}',
         redirectUri: '{redirectUri}',
+        redirect_uri: '{redirect_uri}',
+        storage: '{storage}',
+        autoRenew: '{autoRenew}',
+        auto_renew: '{auto_renew}',
         tokenManager: {
           storage: '{storage}',
-          autoRenew: '{autoRenew}'
+          autoRenew: '{autoRenew}',
         }
       });
     });
+
+    it('Allows passing extra config to tokenManager', () => {
+      const passedConfig = {
+        storage: '{storage}',
+        autoRenew: '{autoRenew}',
+        tokenManager: {
+          secure: true
+        }
+      }
+
+      expect(buildConfigObject(passedConfig)).toEqual({
+        storage: '{storage}',
```

Comment:
`storage` and `autoRenew` should not belong on top-level but to `tokenManager` only

Reply:
I am preserving legacy behavior, where these specific parameters were hoisted down into the `tokenManager` section. The side-effect is that there are 'duplicate' values at the top-level, but this does not cause any negative behavior.